isis/src/control/apps/cnetbin2pvl/cnetbin2pvl.xml

```diff
@@ -7,8 +7,9 @@
 
   <description>
     <p>
-      This program converts an Isis3 control network file from binary into an 
-    ascii pvl formatted file.
+      This program converts an Isis3 control network file from binary into a
+      utf pvl formatted file. The output pvl will be written as the latest Pvl version.

```

Comment:
I think it's utf8 specifically? Maybe mention that. Also, could say human readable